test/support/timeout.rb

```diff
@@ -15,4 +15,5 @@ def capture_exceptions(*args, &block)
     end
   end
 end
-Minitest::Test.prepend TimeoutEveryTestCase
+
+Minitest::Test.prepend TimeoutEveryTestCase unless ENV['SKIP_TIMEOUT']
```

Comment:
Hum... something failed here in the rebase with master, this shouldn't be here. It was removed in master.